drivers/mcp41010.c

```diff
@@ -0,0 +1,58 @@
+#include "jd_drivers.h"
+#include "jd_services.h"
+#include "board.h"
+
+#define MCP_41010_WRITE_DATA 0x10
+#define MCP_41010_SHUTDOWN 0x20
+#define MCP_41010_POT_CHAN_SEL_0 0x01
+#define MCP_41010_POT_CHAN_SEL_1 0x02
+
+static uint8_t mcp_41010_data[] = {0, 0};
+
+static volatile int tx_done = 0;
+static void done_handler(void) {
+    tx_done = 1;
+}
+
+static void spi_tx(void) {
+    pin_set(PIN_CS, 0);
+
+    target_wait_us(100);
+    tx_done = 0;
+    dspi_tx(mcp_41010_data, 2, done_handler);
+    while (tx_done == 0)
+        ;
+    target_wait_us(100);
+
+    pin_set(PIN_CS, 1);
+}
+
+static void mcp41010_set_wiper(uint8_t channel, uint8_t wiper_value) {
+    // mcp41010 only has two channels.
+    JD_ASSERT(channel < 2);
```

Comment:
is the value 0-255 for 0-100% of voltage? if so, maybe use uint16 or 32 and ignore low bits? in case there is a better one in future